src/autoscaler/eventgenerator/generator/appEvaluationManager.go

```diff
@@ -54,31 +43,25 @@ func (a *AppEvaluationManager) Stop() {
 	a.logger.Info("stopped")
 }
 
-func (a *AppEvaluationManager) doEvaluate() {
+func (a *AppEvaluationManager) start() {
 	ticker := a.cclock.NewTicker(a.evaluateInterval)
 	defer ticker.Stop()
 	for {
 		select {
 		case <-a.doneChan:
 			return
 		case <-ticker.C():
-			a.evaluate()
+			a.setTriggers(a.getTriggers())
```

Comment:
triggers := a.getTriggers(a.GetPolicies())

getTriggers is lifted from the Aggregator.